Tests/web3swiftTests/localTests/TransactionsTests.swift

```diff
@@ -195,7 +195,7 @@ class TransactionsTests: XCTestCase {
             // check that we recovered the address correctly
             XCTAssertEqual(jsonTxn.sender!.address, expectedAddress.address, "Recovered Address Mismatch")
         } catch {
-            
```

Comment:
This file was added into PR by accident but since there are only empty line changes it doesn't block.